README.md

```diff
@@ -1,3 +1,12 @@
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+![Python](https://img.shields.io/badge/python-3.9-blue.svg)
+![Python](https://img.shields.io/badge/python-3.10-blue.svg)
+<!-- [![PyPI version](https://badge.fury.io/py/sctk.svg)](https://badge.fury.io/py/sctk) -->
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active) 
+![CI](https://github.com/slobentanzer/sctk/actions/workflows/ci-cd.yaml/badge.svg)
```

Comment:
Given realistic possibilities, link to https://github.com/slobentanzer/sctk/actions/workflows/ci-cd.yaml here?

Reply:
done